src/main/java/core/basesyntax/strategy/impl/BalanceOperationImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceOperationImpl implements OperationHandler {
+
+    @Override
+    public void applyOperation(FruitTransaction transaction) {
+        if (transaction.getQuantity() < MIN_QUANTITY) {
```

Comment:
The `MIN_QUANTITY` constant is used without being declared. You should define this constant in your class. For example, you could create a private static final int MIN_QUANTITY = 0; at the class level.